src/stylesheets/App.css

```
@@ -95,6 +95,7 @@ a:hover {
 
 .quiz-text {
   display: flex;
+  padding: 0 10px 0 10px;
```

small thing, but let's cut down on repetition here

```suggestion
  padding: 0 10px;
```